stroyprombeton/tests/tests_selenium_admin.py

```diff
@@ -347,19 +345,30 @@ def open_filters(self):
 
     def check_filters_and_table_headers_equality(self):
         """TE filters and table headers text should be equal."""
-        filters = self.browser.find_elements_by_class_name('js-sortable-item')
 
-        for index, item in enumerate(filters):
-            filter_text = item.text.lower().replace(':', '')
+        def get_label(element):
+            return element.text.strip().lower().replace(':', '')
+
+        # @todo #317:60m Fix missing "Tags" field selenium admin tests.
+        #  It not appears as table column when selected in the list with fields.
+        def exclude_tag(label: str):
+            return label != 'tags'
+
+        fields = self.browser.find_elements_by_class_name('js-sortable-item')
+        filter_field_labels = filter(exclude_tag, (get_label(f) for f in fields))
+
+        for index, filter_label in enumerate(filter_field_labels):
             # STB call category "раздел", but refarm call it "категория"
-            if 'раздел' in filter_text.lower() or 'вес' in filter_text.lower():
+            if 'раздел' in filter_label or 'вес' in filter_label:
```

Comment:
Move these to `exclude_tag` function

Reply:
@artemiy312 , we can't do it, because index should be iterated over this field labels.
And exclude_tag is a temporary function. It'll be alive until the pdd subtask above will not be resolved

Reply:
@duker33 
>we can't do it

we can:
```python
[
    i, label
    for i, label in enumerate(map(get_label, fields))
    if exclude_tag(label)
]
```
I find it clearer. Up to you

>exclude_tag is a temporary function

We can remove only `label not in ['tags']`

Reply:
our for is with body, but not comprehensive.
We can't use inline if in for with body as i see

This code leads to syntax error
```
for i, f in enumerate(fields) if not exclude(f):
    ...
```